app/controllers/event_sessions_controller.rb

```diff
@@ -26,7 +26,9 @@ def index
 
   def show
     @event_session = EventSession.find(params[:id])
-
+    @event_session_comments = EventSessionComment.where(event_session_id: params[:id])
```

Comment:
Use `@event_session.event_session_comments` in view. Keep the controller as RESTful as possible. Here we are dealing with `event_session` resource, so lets not mix `event_session_comment` here. If you need in view, you can also use model associations like:

`@event_sessions.event_session_comments.new`